tests/runtime/strings/wasm_utf16.c

```diff
@@ -0,0 +1,40 @@
+#include <assert.h>
+#include <imports.h>
+#include <exports.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+
+char16_t BASIC_STRING[] = u"latin utf16";
+// 🚀 = 0xD83D 0xDE80
+// 𠈄 = 0xD840 0xDE04
+// 𓀀 = 0xD80C 0xDC00
+char16_t UNICODE_STRING[] = { 0xD83D, 0xDE80, 0xD83D, 0xDE80, 0xD83D, 0xDE80, ' ', 0xD840, 0xDE04, 0xD80C, 0xDC00 };
+char16_t STR_BUFFER[500];
+
+void assert_str(imports_string_t* str, char16_t* expected, size_t expected_len) {
+  assert(str->len == expected_len);
+  assert(memcmp(str->ptr, expected, 11) == 0);
```

Comment:
I think the 11 here may be a typo

Reply:
I also think that the length needs to be multiplied by 2 to compare the full byte size of the string?

Reply:
Thanks, yes this was left in from testing.